### PR TITLE
MEN-3756: Document why `_INCONSISTENT` is appended to software name.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: 201.Troubleshooting/03.Mender-Client/docs.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ If these mismatch, then you need to update `/etc/mender/server.crt` on your clie
 You can do this manually for testing purposes, and you should
 [include the certificates in your Yocto Project build](../../04.Artifacts/10.Yocto-project/03.Building-for-production/docs.md#including-the-client-certificates).
 
+
+## The Current Software installed on my device has `_INCONSISTENT` appended to it
+
+The `_INCONSISTENT` suffix is appended to the software name on a device when the last Artifact deployment failed, and either the rollback also failed, or the particular Artifact has no rollback capability. As the name implies, in this case the device is in an inconsistent state, somewhere between two known states. In this case the deployment log of the last deployment may provide more information about what went wrong, and whether there is cause for concern.
```
Comment (mirzak) [inline 201.Troubleshooting/03.Mender-Client/docs.md:125]: `the particular Artifact has no rollback capability` -> `the particular Update Module...`?

Comment (kacf) [reply 201.Troubleshooting/03.Mender-Client/docs.md]: Fixed!

File: 201.Troubleshooting/03.Mender-Client/docs.md [hunk 2]
```diff
@@ -119,6 +119,11 @@ If these mismatch, then you need to update `/etc/mender/server.crt` on your clie
 You can do this manually for testing purposes, and you should
 [include the certificates in your Yocto Project build](../../04.Artifacts/10.Yocto-project/03.Building-for-production/docs.md#including-the-client-certificates).
 
+
+## The Current Software installed on my device has `_INCONSISTENT` appended to it
```
Comment (lluiscampos) [inline 201.Troubleshooting/03.Mender-Client/docs.md:123]: I don't think wee need to capitalize Current Software

```suggestion
## The current software installed on my device has `_INCONSISTENT` appended to it
```

Comment (lluiscampos) [reply 201.Troubleshooting/03.Mender-Client/docs.md:123]: But is this passed @mirzak check maybe there is a reason for it...

Comment (kacf) [reply 201.Troubleshooting/03.Mender-Client/docs.md:123]: This was intentional to match the exact string in the GUI. We are doing this in other places too (such as the capitalized RELEASES), so I think this should stay.

Comment (lluiscampos) [reply 201.Troubleshooting/03.Mender-Client/docs.md:123]: Makes sense. Thanks.

**Comment by kacf** [comment]
Cherry-picked to 2.3.x and 2.4.x as well.

**Pull request merged**